package.json

```diff
@@ -5,20 +5,25 @@
   "dependencies": {
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
-    "@mui/material": "^5.11.7",
+    "@mui/icons-material": "^5.11.0",
+    "@mui/material": "^5.11.8",
+    "@mui/styled-engine-sc": "^5.11.0",
     "@popperjs/core": "^2.11.6",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.3.2",
     "bootstrap": "^5.2.3",
+    "install": "^0.13.0",
     "jquery": "^3.6.3",
+    "npm": "^9.4.2",
```

Comment:
Do you actually want `npm` and `install` in here or were these typo's that got added by accident?